WordPressAuthenticator/Model/WordPressComSiteInfo.swift

```diff
@@ -24,6 +24,8 @@ public class WordPressComSiteInfo {
     /// URL of the Site's Blavatar.
     ///
     public let icon: String
+    
+    public let isWPCom: Bool
```

Comment:
For consistency could we add a comment describing the property?  Its pretty obvious but 🤷‍♂.